### PR TITLE
Add the consolidated release notes for 2.5.0

### PR DESCRIPTION
**Pull request by zelinh**
Signed-off-by: Zelin Hao <zelinhao@amazon.com>

### Description
Add the consolidated release notes for 2.5.0

### Issues Resolved
Part of #2908 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: release-notes/opensearch-release-notes-2.5.0.md [hunk 1]
```diff
@@ -0,0 +1,680 @@
...
+* With this release, Security Analytics for OpenSearch and OpenSearch Dashboards is generally available, offering a number of tools to help users protect their data and infrastructure.
+* You can build multi-layer maps from multiple data sources, combining data from different indexes into a single visualization to identify correlations and gain insights into geospatial data.
+* New Debian distributions let you deploy OpenSearch and OpenSearch Dashboards directly on servers running Debian-based Linux distributions.
+* Administrators can now view the health of their cluster at the awareness attribute level when shard allocation awareness is configured.
+* You can now search your rollup indexes using query string search queries.
+
+### Experimental features
+
+OpenSearch 2.5.0 includes the following experimental features. Experimental features are disabled by default. For instructions on how to enable them, see the version history (https://opensearch.org/docs/latest/version-history/) page which includes links to the documentation.
+
```
Comment (jhmcintyre) [inline release-notes/opensearch-release-notes-2.5.0.md:28]: remove blank line between bullets

File: release-notes/opensearch-release-notes-2.5.0.md [hunk 2]
```diff
@@ -0,0 +1,680 @@
...
+OpenSearch and OpenSearch Dashboards 2.5.0 includes the following feature, enhancement, bug fixes, infrastructure, documentation, maintenance and refactoring updates.
+
+OpenSearch [Release Notes](https://github.com/opensearch-project/OpenSearch/blob/main/release-notes/opensearch.release-notes-2.5.0.md).
+
+OpenSearch Dashboards [Release Notes](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/release-notes/opensearch-dashboards.release-notes-2.5.0.md).
+
+## Release Highlights
+
+The OpenSearch 2.5.0 release adds new tools and enhancements to help you advance your search, analytics, and observability workloads. This release includes the software’s first Debian distribution and first administrative user interface, along with support for multi-layered maps and the ability to analyze traces in the Jaeger schema. The release also includes Indexing and search improvements for Lucene-based k-NN search functionality, and Security Analytics tools are now generally available. Following are some highlights for this release.
```
Comment (jhmcintyre) [inline release-notes/opensearch-release-notes-2.5.0.md:13]: indexing should be lowercase

File: release-notes/opensearch-release-notes-2.5.0.md [hunk 3]
```diff
@@ -0,0 +1,680 @@
...
+
+
+## FEATURE
+
+### OpenSearch Dashboards Maps
+* Add tooltip for hover ([#132](https://github.com/opensearch-project/dashboards-maps/pull/132))
+* Introduce tooltip fields to document layer specification ([#124](https://github.com/opensearch-project/dashboards-maps/pull/124))
+* Add support for WKT format ([#165](https://github.com/opensearch-project/dashboards-maps/pull/165))
+* Add global time filter bar to maps ([#131](https://github.com/opensearch-project/dashboards-maps/pull/131))
+* Support custom layer to Maps ([#150](https://github.com/opensearch-project/dashboards-maps/pull/150))
```
Comment (jhmcintyre) [inline release-notes/opensearch-release-notes-2.5.0.md:40]: "for maps" (make maps lowercase to be consistent with rest of notes)

File: release-notes/opensearch-release-notes-2.5.0.md [hunk 4]
```diff
@@ -0,0 +1,680 @@
...
+* Support geo_shape visualization in documents layer ([#111](https://github.com/opensearch-project/dashboards-maps/pull/111))
+* Introduce saved object plugin into maps plugin ([#67](https://github.com/opensearch-project/dashboards-maps/pull/67))
+* Add new layer functions for opensearch map layer ([#66](https://github.com/opensearch-project/dashboards-maps/pull/66))
```
Comment (jhmcintyre) [inline release-notes/opensearch-release-notes-2.5.0.md:43]: OpenSearch

File: release-notes/opensearch-release-notes-2.5.0.md [hunk 5]
```diff
@@ -0,0 +1,680 @@
...
+* Add layer panel component ([#51](https://github.com/opensearch-project/dashboards-maps/pull/51))
+* Add base map layer functions ([#62](https://github.com/opensearch-project/dashboards-maps/pull/62))
+* Query with geo bounding box ([#148](https://github.com/opensearch-project/dashboards-maps/pull/148))
+* Display search filters in map layer config panel ([#130](https://github.com/opensearch-project/dashboards-maps/pull/130))
+
+
+### OpenSearch Observability Dashboards Plugin
+* Add log pattern table ([#1187](https://github.com/opensearch-project/observability/pull/1187))
+* Implementing search feature ([#1286](https://github.com/opensearch-project/observability/pull/1286))
+* Traces jaeger ([#150](https://github.com/opensearch-project/dashboards-observability/pull/150))
```
Comment (jhmcintyre) [inline release-notes/opensearch-release-notes-2.5.0.md:54]: suggest "Add support for raw Jaeger schema data"

File: release-notes/opensearch-release-notes-2.5.0.md [hunk 6]
```diff
@@ -0,0 +1,680 @@
...
+### OpenSearch SQL
+* Add low-level create table and table exists API ([#834](https://github.com/opensearch-project/sql/issues/834))
+* Add time window and window assigner ([#950](https://github.com/opensearch-project/sql/issues/950))
+* Add valueOf() to Expression ([#1055](https://github.com/opensearch-project/sql/issues/1055))
+* Add Statement, QueryExecution and QueryManager ([#845](https://github.com/opensearch-project/sql/issues/845))
+* Add Streaming Source Impl ([#994](https://github.com/opensearch-project/sql/issues/994))
+* Add watermark generator ([#959](https://github.com/opensearch-project/sql/issues/959))
+* Add stream context and window trigger ([#958](https://github.com/opensearch-project/sql/issues/958))
+* Add micro batch streaming execution ([#1044](https://github.com/opensearch-project/sql/pull/1044))
+* Add Streaming Plan Impl ([#1068](https://github.com/opensearch-project/sql/issues/1068))
```
Comment (jhmcintyre) [inline release-notes/opensearch-release-notes-2.5.0.md:185]: Implementation

File: release-notes/opensearch-release-notes-2.5.0.md [hunk 7]
```diff
@@ -0,0 +1,679 @@
+# OpenSearch and OpenSearch Dashboards 2.5.0 Release Notes
+
+## Release Details
+
+OpenSearch and OpenSearch Dashboards 2.5.0 includes the following feature, enhancement, bug fixes, infrastructure, documentation, maintenance and refactoring updates.
```
Comment (dblock) [inline release-notes/opensearch-release-notes-2.5.0.md:5]: Fix or fixes?

File: release-notes/opensearch-release-notes-2.5.0.md [hunk 8]
```diff
@@ -0,0 +1,679 @@
...
+OpenSearch [Release Notes](https://github.com/opensearch-project/OpenSearch/blob/main/release-notes/opensearch.release-notes-2.5.0.md).
+
+OpenSearch Dashboards [Release Notes](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/release-notes/opensearch-dashboards.release-notes-2.5.0.md).
+
+## Release Highlights
+
+The OpenSearch 2.5.0 release adds new tools and enhancements to help you advance your search, analytics, and observability workloads. This release includes the software’s first Debian distribution and first administrative user interface, along with support for multi-layered maps and the ability to analyze traces in the Jaeger schema. The release also includes indexing and search improvements for Lucene-based k-NN search functionality, and Security Analytics tools are now generally available. Following are some highlights for this release.
+
+### New features
+
```
Comment (dblock) [inline release-notes/opensearch-release-notes-2.5.0.md:16]: Capitalize features

File: release-notes/opensearch-release-notes-2.5.0.md [hunk 9]
```diff
@@ -0,0 +1,679 @@
...
+* Correct ppl learn more link ([#154](https://github.com/opensearch-project/dashboards-observability/pull/154))
+* Fixed PPL Error in containers \& added tool tip to date picker ([#179](https://github.com/opensearch-project/dashboards-observability/pull/179))
+* Fix explorer dark mode issue and restructure scss ([#157](https://github.com/opensearch-project/dashboards-observability/pull/157))
+* Few fixes regarding issues for visualization rendering ([#186](https://github.com/opensearch-project/dashboards-observability/pull/186))
+* Fix bug with overriding patterns ([#1298](https://github.com/opensearch-project/observability/pull/1298))
+* QS to 6.5.3 ([#1335](https://github.com/opensearch-project/observability/pull/1335))
+
+
+### OpenSearch Query Workbench
+* Bump json5 from 2.2.1 to 2.2.3 ([#20](https://github.com/opensearch-project/dashboards-query-workbench/pull/20))
```
Comment (dblock) [inline release-notes/opensearch-release-notes-2.5.0.md:259]: I don’t think we include these everywhere, so remove this and similar?

**Review by dblock — COMMENTED** [review]
Nits

File: release-notes/opensearch-release-notes-2.5.0.md (hunk 6 above)
Comment (zelinh) [reply release-notes/opensearch-release-notes-2.5.0.md]: Thanks. @jhmcintyre I have updated based on your review. Please help take another look then. Thanks again!

File: release-notes/opensearch-release-notes-2.5.0.md (hunk 7 above)
Comment (zelinh) [reply release-notes/opensearch-release-notes-2.5.0.md]: It looks like we have been using `bug fixes` through all previous notes. I would like to keep this as what it was.

File: release-notes/opensearch-release-notes-2.5.0.md (hunk 8 above)
Comment (zelinh) [reply release-notes/opensearch-release-notes-2.5.0.md:8]: Thanks! @dblock Just updated.

**Review by bbarani — APPROVED** [review]

**Pull request merged**